spec/requests/collections_spec.rb

```diff
@@ -17,4 +17,15 @@
       expect(response).to have_http_status(:ok)
     end
   end
+
+  describe "GET large user collection show" do
+    it "returns the proper article count and text for a large collection" do
```

Comment:
_Quick, non-blocking suggestion_:
```suggestion
    it "returns the proper article count and text for a large collection", :aggregate_failures do
```
For context, [here's](https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures) a concise explanation of what `:aggregate_failures` does!

Reply:
I noticed `aggregate_failures` in your previous message, but I didn't take the time to look into what it does. Thanks for linking the documentation! 😄

Reply:
Absolutely! 😊